fakenet/listeners/IRCListener.py

```diff
@@ -159,6 +159,11 @@ def irc_DEFAULT(self, cmd, params):
         self.server.logger.info('Client issued an unknown command %s %s', cmd, params)      
         self.irc_send_server("421", "%s :Unknown command" % cmd)          
 
+    def irc_WHOIS(self, cmd, params):
+        output = f"ircname : {self.realname}\nhostname : {self.user}\nserver : {self.server}\nmodes : {self.mode}"
```

Comment:
what reference did you use for the output format for WHOIS command?

Reply:
As of now, I have taken it from the original `WHOIS` command output from the `irssi` client. The output is attached above as an image(1st image in PR). The reply format has to be refined further to match the exact response.

Reply:
We want the format to be the standard format used by WHOIS command replies. Refer https://www.ietf.org/rfc/rfc1459.txt.

Reply:
Got it, will follow up with the ideal standard